content/applications/inventory_and_mrp/inventory/shipping_receiving/reservation_methods/before_scheduled_date.rst

```diff
@@ -0,0 +1,118 @@
+this specific product to five days after the creation date of the sales order.
+
+.. image:: before_scheduled_date/before-scheduled-date-customer-lead-time.png
+   :align: center
+   :alt: Product form with customer lead time set in Inventory tab.
+
+Workflow
+========
+
+To see the *Before scheduled date* reservation method in action, create a new :abbr:`SO (sales
+order)` by navigating to :menuselection:`Sales app --> New`.
+
+Add a customer in the :guilabel:`Customer` field, then, in the :guilabel:`Order Lines` tab, click
+:guilabel:`Add a product`, and select a product to add to the quotation from the drop-down menu.
+Finally, in the :guilabel:`Quantity` column, adjust the desired quantity of the product to sell.
+
+Once ready, click :guilabel:`Confirm` to confirm the sales order.
+
+Click the green :guilabel:`📈 (forecast graph)` icon on the product line to reveal the product's
+:guilabel:`Availability` tooltip. This tooltip reveals the reserved number of units for this order.
+Because the reservation method is set to *Before scheduled date*, the :guilabel:`Reserved` quantity
+reads `0 Units`.
+
+However, below that quantity reads `Available in stock`. This is because the quantity is available,
+but the scheduled date is five days from the order date. Since reservation isn't until two days
+before the scheduled delivery, it will not reserve the products until then.
+
+.. image:: before_scheduled_date/before-scheduled-date-availability-tooltip.png
+   :align: center
+   :alt: Confirmed sales order with product availability tooltip selected.
+
+Click the :guilabel:`Delivery` smart button to see the delivery order form.
+
+On the :guilabel:`Delivery Order` form, the status in the :guilabel:`Product Availability` field is
+listed as `Available`, in yellow text instead of green. This is because there is sufficient stock on
+hand for this order, but no quantity has been reserved yet.
+
+Note the :guilabel:`Scheduled Date` field above the :guilabel:`Product Availability` field displays
+the date five days from the order creation date. This indicates that the products will not be
+reserved until three days from today's date (two days before the scheduled delivery date).
+
+.. image:: before_scheduled_date/before-scheduled-date-delivery-order-form.png
+   :align: center
+   :alt: Delivery order form with product availability and reserved quantity.
+
+In the :guilabel:`Operations` tab on the :guilabel:`Product` line, the numbers in the
+:guilabel:`Demand` column and the :guilabel:`Quantity` column do not match (in this case, the
+:guilabel:`Demand` column lists `10.00`, while the :guilabel:`Quantity` column lists `0`.
+
+.. tip::
+   If the product(s) in the :abbr:`SO (sales order)` should be reserved sooner than the scheduled
+   reservation date, the reservation can be manually "overridden". To manually reserve the products
+   sooner than scheduled, click :guilabel:`Check Availability` at the top of the form.
```

Comment:
Yeah, I just thought it would be useful to indicate that it can apply to both orders with 1 product, and orders with multiple. But I'll stick to the rules here, Suggestion Implemented ✅